heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
```

Comment:
Similar to this file, can you add a sample for heron.aurora - if somebody wants to use docker uploader. You could add this under the directory heron/config/src/yaml/aurora - with the file name heron.aurora.docker.